File: packages/pensopay_android/android/src/main/kotlin/com/flutter/pensopay/networking/pensopayapi/pensopaylink/models/PPSubscription.kt
package com.flutter.pensopay.networking.pensopayapi.pensopaylink.models

import org.json.JSONObject
import kotlin.collections.HashMap

class PPSubscription: JSONObject() {

    var id: Int = 0
    var subscription_id: String = ""
    var amount: Int = 0
    var currency: String = ""
    var state: String = ""
    var description: String = ""
    var callback_url: String? = null
    var variables: Map<String, Any>? = null
    var created_at: String = ""
    var updated_at: String = ""

}

File: packages/pensopay_ios/ios/Classes/Networking/Pensopay API/Pensopay Link/Models/PPSubscription.swift
//
//  PPSubscription.swift
//  PensopaySDK
//
//  Created on 07/26/2022
//  Copyright © 2022 Pensopay. All rights reserved.
//

import Foundation

public class PPSubscription: Codable {
    
    // MARK: - Properties

    public var id: Int
    public var subscription_id: String
    public var amount: Int
    public var currency: String
    public var state: String
    public var description: String
    public var callback_url: String?
    public var variables: Array<String>?
    public var created_at: String
    public var updated_at: String

}
